lib/builders/latexmk.js

```diff
@@ -4,6 +4,8 @@ import childProcess from 'child_process'
 import path from 'path'
 import Builder from '../builder'
 
+const LATEX_PATTERN = /^latex|u?platex$/
```

Comment:
Listing `latex` here is probably not needed. `latex` is currently symlinked to `pdflatex` as are most LaTeX engines that are listed as DVI producing engines. So far the only other one that I have been able to discover that cannot produce PDF is `lamed` which I believe is not really used anymore.